src/Evolution/Systems/GrMhd/ValenciaDivClean/Characteristics.hpp

```diff
@@ -112,6 +113,22 @@ std::array<DataVector, 9> characteristic_speeds(
     const EquationsOfState::EquationOfState<true, ThermodynamicDim>&
         equation_of_state) noexcept;
 
+template <size_t ThermodynamicDim>
+void characteristic_speeds(
```

Comment:
I'm confused.  Am I reading this correctly that this function already existed but was missing a declaration in the header file?

Reply:
Yes, before it was only used internally from the not not_null version.